OpenRA.Mods.TS/Activities/VoxelHarvesterDockSequence.cs

```diff
@@ -17,24 +18,52 @@ namespace OpenRA.Mods.TS.Activities
 	public class VoxelHarvesterDockSequence : HarvesterDockSequence
 	{
 		readonly WithVoxelUnloadBody body;
+		readonly WithHarvesterUnloadOverlay spriteOverlay;
```

Comment:
That will conflict with the already existing https://github.com/OpenRA/OpenRA/wiki/Traits#withdockingoverlay